### ISSUE TITLE
Bug: No detail for invoices - got error related to PDF [cause identified].

### ISSUE THREAD

**Issue by mdeweerd** [Dolibarr/dolibarr#7367]
After upgrade from 5.0.3 to 6.0.0 .

PHP/Imagick Error logged by web server:

```
2017-09-10 17:26:05: (mod_fastcgi.c.2676) FastCGI-stderr: PHP Fatal error:  Uncaught exception 'ImagickException' with message 'Postscript delegate failed `/var/dolibardocs/facture/FA1708-0273/FA1708-0273.pdf': No such file or directory @ error/pdf.c/ReadPDFImage/677' in /home/www/DOLIBARR/dolibarr-all/htdocs/core/lib/files.lib.php:1577
Stack trace:
#0 /home/www/DOLIBARR/dolibarr-all/htdocs/core/lib/files.lib.php(1577): Imagick->readimage('/var/dolibardoc...')
#1 /home/www/DOLIBARR/dolibarr-all/htdocs/core/lib/functions.lib.php(1241): dol_convert_file('/var/dolibardoc...', 'png', '/var/dolibardoc...')
#2 /home/www/DOLIBARR/dolibarr-all/htdocs/compta/facture/card.php(3116): dol_banner_tab(Object(Facture), 'ref', '<a href="/dolib...', 1, 'facnumber', 'ref', '<div class="ref...', '', 0, '', '')
#3 {main}
  thrown in /home/www/DOLIBARR/dolibarr-all/htdocs/core/lib/files.lib.php on line 1577

```

The PDF file exists and the server has access (www-data):
```
root@www facture/FA1708-0273# ls -l /var/dolibardocs/facture/FA1708-0273/FA1708-0273.pdf
-rw-rw-r-- 1 www-data www-data 167990 Aug 29 20:12 /var/dolibardocs/facture/FA1708-0273/FA1708-0273.pdf
```

**Comment by mdeweerd** [comment]
May be related to: https://github.com/Dolibarr/dolibarr/issues/7352 .  The behavior from a user point of view is the same, but no technical error is mentionned it that other issue.

**Comment by mdeweerd** [comment]
Changed code (added try/catch) in files.lib.php - the tabs have their content now, the root cause is not fixed yet.
```
        if (class_exists('Imagick'))
        {
                $image=new Imagick();
try {
                $ret = $image->readImage($fileinput);
} catch(Exception $e) {
   return 0;
}
```

**Comment by mdeweerd** [comment]
I fixed it by installing ghostscript on my debian platform:

` apt-get install ghostscript`

I am leaving this issue open as I think that this error should not be causing a fatal error in dolibarr + dolibarr could suggest installing ghostscript.

**Comment by eldy** [comment]
Fixed for 6.0.1
